gateway/main.go

```diff
@@ -64,7 +64,7 @@ func Run() {
 	}
 
 	idProvider := idp.NewProvider(apiURL)
-	grpcURL := os.Getenv("GRPC_URL")
+	grpcURL := appconfig.Get().GrpcURL()
 	if grpcURL == "" {
 		scheme := "grpcs"
 		if appconfig.Get().ApiScheme() == "http" {
```

Comment:
Remove this condition, it will never be triggered

Reply:
you mean the whole condition, beginning at line 68, right?

Reply:
Yes